keymanager/src/crypto/types.rs

```diff
@@ -0,0 +1,110 @@
+use std::{default::Default, vec};
```

Comment:
As above, neither should be needed due to std prelude being imported by default.